src/products/test/integ/test-products.py

```diff
@@ -0,0 +1,142 @@
+import jsonschema
+import requests
+import json
+import os
+
+
+products_api_url = os.getenv('PRODUCTS_API_URL')
+cwd = os.path.dirname(os.path.abspath(__file__))
+wildcards = {
+    ":product_id": os.getenv('TEST_PRODUCT_ID'),
+    ":category_name": os.getenv('TEST_CATEGORY_ID'),
+    ":category_id": os.getenv('TEST_CATEGORY_ID')
+}
+
+
+def full_request_url(text):
```

Comment:
these functions are very generic, have a library somewhere instead of in each service?

Reply:
I've moved these methods to their own module so we can reuse them for the other services.